config/config.go

```diff
@@ -254,6 +258,16 @@ func LoadConfig(p string) error {
 		}
 	}
 
+	kb := c.MaxResponseSize * 1024 // to kilobyte
+	if kb == 0 {
+		c.MaxResponseSize = MaxResponseSize
+	} else if kb > MaxResponseSize {
+		log.Warnf("maximum response size of %dkb too large, using default value of 50kb", c.MaxResponseSize)
```

Comment:
@danvixent This error message seems off.

Reply:
hm, c.MaxResponseSize is passed in from config in kb terms. How do you suggest we make it better.

Reply:
I mean It should be `log.Warnf("maximum response size of %dkb is too large, using default value of %dkb", kb, c.MaxResponseSize)`

Reply:
Nah, kb is the resolved into bytes equivalent of c.MaxResponseSize, and MaxResponseSize(the const) is also in bytes.
We want to print it in kb

Reply:
Got it. You can change the `50kb` to use the const -- `MaxResponseSize` though.

Reply:
done